.github/workflows/lint-check.yml

```
@@ -0,0 +1,102 @@
+          
+      - name: 📦 Install pnpm
+        uses: pnpm/action-setup@v3
+        with:
+          version: '10'
+          
+      - name: 📥 Install dependencies
+        run: pnpm install --frozen-lockfile
+        
+      - name: 🧪 ESLint Check
+        id: eslint
+        run: |
```

actionlint (1.7.7)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)